src/guide/1/pubsub/android/index.md

```diff
@@ -88,9 +88,9 @@ We have now programmed the subscription side of the test. Let's move on to the *
 
 ## Publishing a Document
 
-Now let's move on to the publish side of the test. Here we will publish a document that contains the `message` field. When Kuzzle receives this message, it will detect that there is a subscriber listening for such messages and will send it to these subscribers, in this case to our Android App.
+Now let's move on to the publish side of the test. Here we will publish a document containings the `message` field. When Kuzzle receives this message, it will detect that there is a subscriber listening for such messages and will send it to these subscribers, in this case to our Android App.
```

Comment:
```suggestion
Now let's move on to the publish side of the test. Here we will publish a document containing the `message` field. When Kuzzle receives this message, it will detect that there is a subscriber listening for such messages and will send it to these subscribers, in this case to our Android App.
```